enqueue/enqueue-bundle/0.3/etc/packages/test/enqueue.yaml

```diff
@@ -0,0 +1,6 @@
+enqueue:
+    transport:
+        default: 'null'
+        'null': ~
```

Comment:
This is extremely confusing to me: a "default transport" which is a null string ... which then is a "null transport" with a null value. Can we think a better approach for this? A new config option for tests maybe? Thanks!